components/content/src/page.rs

```diff
@@ -115,6 +115,10 @@ impl Page {
         page.word_count = Some(word_count);
         page.reading_time = Some(reading_time);
 
+        if !page.meta.render {
```

Comment:
Do we want to still process slugs etc even if we don't render things?

Reply:
Hi Keats,

No, my intention was to stop processing of slugs, paths and sitemap entries if `meta.render` is set. 

I assumed that, since `parse()` returns at this point, the job is done but your question makes me doubt that :) I'm not fluent in the codebase (or Rust for that matter) so if there are any other places where I should do the plugging please let me know.

Reply:
It's mostly a question of whether we want a fully populated page, except it's not rendered which I think is better?

Reply:
I think it's probably better to have the fully processed pages so if you query it in a template, you have all the info you need